internal/config/drive/drive.go

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2015-2023 MinIO, Inc.
+//
+// This file is part of MinIO Object Storage stack
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package drive
+
+import (
+	"sync"
+	"time"
+
+	"github.com/minio/minio/internal/config"
+	"github.com/minio/pkg/v2/env"
+)
+
+// DefaultKVS - default KVS for drive
+var DefaultKVS = config.KVS{
+	config.KV{
+		Key:   MaxTimeout,
+		Value: "",
+	},
+}
+
+// Config represents the subnet related configuration
+type Config struct {
+	// Flag indicating whether callhome is enabled.
+	MaxTimeout time.Duration `json:"maxTimeout"`
+}
+
+var mutex sync.RWMutex
```

Comment:
This seems misplaced.

Add the mutex to the data it is protecting. Ie inside the Config. Package level locks are a no-no.